From 0bfe2feaa53f611954722afcd44d3b5cb228cadf Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 26 Feb 2024 14:11:12 +0100
Subject: [PATCH 0664/1695] dts: stm32: default disable TAMP node on STM32MP13x
 boards

Default disable TAMP node so that it's enabled at platform level.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 3570730dcbe3d53e3f00b22a0c2f91959f81ffe6 dts: stm32: default disable TAMP node on STM32MP13x boards]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: If1d11deb24db4fbcb1bca77dc517fe070c3fd093
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/362911
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp131.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/dts/stm32mp131.dtsi b/core/arch/arm/dts/stm32mp131.dtsi
index c21439a28..0527643e4 100644
--- a/core/arch/arm/dts/stm32mp131.dtsi
+++ b/core/arch/arm/dts/stm32mp131.dtsi
@@ -398,6 +398,7 @@
 			interrupts = <GIC_SPI 128 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&rcc RTCAPB>;
 			ranges;
+			status = "disabled";
 
 			nvram: nvram@5c00a100 {
 				compatible = "st,stm32mp15-tamp-nvram";
-- 
2.47.2

